client/webserver/site/src/js/markets.js

```diff
@@ -1313,15 +1313,16 @@ export default class MarketsPage extends BasePage {
     }
     if (!this.validateOrder(order)) return
     const res = await postJSON('/api/trade', req)
-    if (!app.checkResponse(res)) return
-    // If the wallets are not open locally, they must have been opened during
-    // ordering. Grab updated info.
-    const baseWallet = app.walletMap[market.base.id]
-    const quoteWallet = app.walletMap[market.quote.id]
-    if (!baseWallet.open || !quoteWallet.open) {
-      this.balanceWgt.updateAsset(market.base.id)
-      this.balanceWgt.updateAsset(market.quote.id)
+    // If errors display error on confirmation modal.
+    if (!app.checkResponse(res, true)) {
+      page.vErr.textContent = res.msg
+      Doc.show(page.vErr)
+      page.vSubmit.disabled = false
+      return
     }
+    // Hide confirmation modal only on success.
+    Doc.hide(page.forms)
+    page.vSubmit.disabled = false
```

Comment:
Can move `page.vSubmit.disabled = false` up before `if (!app.checkResponse(res, true)) {` and only do it once.

Reply:
Bump. Good to merge after this last thing.

Reply:
The button needs to go gray too if not a spinner.

Reply:
on it now

Reply:
Done